src/plugins/integrations/AWSIntegration.ts

```diff
@@ -209,10 +212,11 @@ class AWSIntegration implements Integration {
               activeSpan.setTag(SpanTags.TOPOLOGY_VERTEX, true);
               activeSpan.setTag(SpanTags.TRIGGER_DOMAIN_NAME, LAMBDA_APPLICATION_DOMAIN_NAME);
               activeSpan.setTag(SpanTags.TRIGGER_CLASS_NAME, LAMBDA_APPLICATION_CLASS_NAME);
+              activeSpan.setTag(AwsS3Tags.OBJECT_NAME, request.params.Key);
             }
           } else if (serviceName === 'lambda') {
             const operationType = LambdaRequestType[operationName];
-            const lambdaName = operationType ? request.params.FunctionName : AWS_SERVICE_REQUEST;
+            const lambdaName = koalas(request.params.FunctionName, AWS_SERVICE_REQUEST);
```

Comment:
If function name is not valid for the request, it is ok to use `AWS_SERVICE_REQUEST` as the default span name, but it should not be used for `AwsLambdaTags. FUNCTION_NAME` tag. So you may handle this by introducing another local variable for span name.